metadata-models/src/main/pegasus/com/linkedin/policy/DataHubPolicyInfo.pdl

```diff
@@ -48,4 +51,11 @@ record DataHubPolicyInfo {
    */
   editable: boolean = true
 
+  /**
+   * Timestamp when the policy was last updated
+   */
+  @Searchable = {
+    "fieldType": "DATETIME"
+  }
+  lastUpdatedTimestamp: optional long
```

Comment:
Thoughts on lastModifiedMs -- only cause timestamp can mean a few formats, also to align with startTimeMs in the ExecutionRequestResult model that we use for sorting ingestion runs. Not a huge deal but let me know what you think

Reply:
Oh I just followed the one added to Operation

Reply:
Found it https://github.com/datahub-project/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/common/Operation.pdl#L18 This is where I got it from. I think we've been using "last updated" in the UI so going to keep it like this